js/src/wcdl/select-control/index.scss

```diff
@@ -54,7 +54,7 @@
 				}
 				.woocommerce-tag__text {
 					padding: 0 $grid-unit-10;
-					border-radius: 12px;
+					border-radius: $grid-unit-15;
```

Comment:
❔ Same as https://github.com/woocommerce/google-listings-and-ads/pull/1578#discussion_r920833917.